cwe_api/cvss_inferencer.py

```
@@ -0,0 +1,37 @@
+    output_decoded = output_decoded[:-1]
+
+    pair_name2letter = {'AV': {'NETWORK': 'N', 'ADJACENT': 'A', 'LOCAL': 'L', 'PHYSICAL': 'P'},
+                        'AC': {'LOW': 'L', 'HIGH': 'H'},
+                        'PR': {'NONE': 'N', 'LOW': 'L', 'HIGH': 'H'},
+                        'UI': {'NONE': 'N', 'REQUIRED': 'R'},
+                        'S': {'UNCHANGED': 'U', 'CHANGED': 'C'},
+                        'C': {'NONE': 'N', 'LOW':'L', 'HIGH': 'H'},
+                        'I': {'NONE': 'N', 'LOW': 'L', 'HIGH':'H'},
+                        'A': {'NONE':'N', 'LOW':'L', 'HIGH':'H'}}
+    
+    cvss_string = 'CVSS:3.1/'
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)